core/controllers/topics_and_skills_dashboard.py

```diff
@@ -140,6 +142,7 @@ def post(self):
         description = self.payload.get('description')
         linked_topic_ids = self.payload.get('linked_topic_ids')
         explanation_dict = self.payload.get('explanation_dict')
+        temp_id = self.payload.get('temp_id')
```

Comment:
Variable name needs more detail. What is this an ID of, conceptually? Does the ID relate to actual entity IDs in any way?

Reply:
Renamed it to 'temp_skill_id'.